cpp/inc/bond/core/transforms.h

```diff
@@ -388,6 +388,12 @@ class RequiredFieldValiadator
         _required = next_required_field<typename schema<T>::type::fields>::value;
     }
 
+    void Base(bool done) const
```

Comment:
Technically, this is a breaking change. Not sure if this needs to be mentioned in changelog.

Reply:
Yes, this should be mentioned in the change log.